spec/controllers/friendships_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper'
+
+describe FriendshipsController do 
+  describe 'GET index' do 
+    it 'sets @friendships to the current users following friendships' do 
+      joe = Fabricate(:user)
+      jane = Fabricate(:user)
+      setup_current_user(joe)
+      friends = Fabricate(:friendship, leader: jane, follower: joe)
+      get :index
+      expect(assigns(:friendships)).to eq([friends])
+    end 
+    
+    it_behaves_like 'require_sign_in' do 
+      let(:action) { get :index }
+    end
+  end
+  
+  describe 'DELETE destroy' do 
+    it_behaves_like 'require_sign_in' do 
+      let(:action) { delete :destroy, id: 4 } 
+    end
+    
+    it 'deletes the friendship if the current user is the follower' do 
+      john = Fabricate(:user)
+      jane = Fabricate(:user)
```

Comment:
I would move these to let `let(:john) { Fabricate(:user) }` your tests are short enough that it is easily readable and it leaves them less cluttered.